Doc/cmus.txt

```diff
@@ -1064,6 +1064,11 @@ icecast_default_charset (ISO-8859-1)
 
 	Note: This is used only if the metadata is not valid UTF-8.
 
+ignore_duplicates (true)
+	Ignore duplicates when adding tracks to the library. A track is a duplicate
+	if it has a matching artist, album, disc number, track number, and track
```

Comment:
I wonder if it might be a good idea to check the length too.

Reply:
On the one hand I'd worry the same track could be +/- 1 second depending on encoder, album edition, etc. At the same time, in my quite tidy library there was just one duplicate: an album with two "0. Bonus Track" that were in fact different bonus tracks. A length check would have caught that.

Overall I don't know which works better for all the real world messy libraries out there. If I go by feelings, the second "0. Bonus Track" _deserved_ to get removed. :grin:

Reply:
> A length check would have caught that.

That's the kind of situation I had in mind.

> real world messy libraries out there

For simplicity, it might be easier to keep it without the length check, and to add an option to force-add tracks.